### DISCUSSION TITLE
Cannot navigate to the PR from pr_id/commits link

### DISCUSSION THREAD

**Discussion by amadejpapez** [community/community#140247] [Mobile]
https://github.com/curl/curl/discussions/13743#discussioncomment-10804150

I stumbled across this comment in the app. It links to this https://github.com/curl/curl/pull/14135/commits. To the commits tab of the PR.

Issue:
I cannot get to the actual PR after clicking on this link in a comment. "Back" bring me back to the comment and there isn't much else displayed.

![image](https://github.com/user-attachments/assets/e3ecf1d8-3909-4799-865a-2d703ce637ec)

I had to Open in Safari and then navigate back there and open in the app that link.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.179.0 (142164528)
Device: iPhone 15 Plus (iOS 18.0)
</details>

**Comment by stevepopovich** [comment]
Hi @amadejpapez thanks for writing in. I will file this request with the team and decide how to deal with this. We will let you know if we end up fixing this!
